eventmesh-common/src/main/java/org/apache/eventmesh/common/config/CommonConfiguration.java

```diff
@@ -29,23 +29,13 @@
     public String eventMeshName = "";
     public String sysID = "5477";
     public String eventMeshConnectorPluginType = "rocketmq";
+    public String eventMeshAclPluginType = "acl";
```

Comment:
eventMeshAclPluginType -> eventMeshSecurityPluginType is better

Reply:
OK， I will adjust it.